src/Core/Types/Utilities/XmlDocumentationExtensions.cs

```diff
@@ -0,0 +1,514 @@
+using System;
```

Comment:
This file is effectively a copy-paste of [`XmlDocumentationExtensions`](https://github.com/RicoSuter/NJsonSchema/blob/master/src/NJsonSchema/Infrastructure/XmlDocumentationExtensions.cs) in NJsonSchema, and adapted to use the correct .NET Standard APIs as opposed to dynamic invocation through reflection.

Reply:
I have rewritten it to be more maintainable. I am still working on it.